bitnami/metallb/Chart.yaml

```diff
@@ -1,8 +1,8 @@
-annotations:
-  category: Infrastructure
 apiVersion: v2
-appVersion: 0.9.5
+name: metallb
 description: The Metal LB for Kubernetes
+version: 1.0.2
```

Comment:
We should bump the major version. Also, add a note in the README about the concluded changes and how to upgrade the chart to this new version

Reply:
It is bumped.
I don't introduced breaking changes.
The breaking changes were the changes in RBAC naming.
Everything else is the same its just restructuring.

Reply:
I could also bump it to 2.0.0.
I don't think that it breaks much. It' just restructured into more files.
And logically separated.
What we can bump is the medium version I think. 
We should first try an upgrade from 1.0.1 to my chart. When that's not a breaking change we can increase this version.
I could also add backwards compatibility for the prometheus part if that is needed.

Reply:
I think the issue is, we cannot say for sure that its intercompatible.

Reply:
Hi @Cellebyte ,
We usually bump major versions if:
- The backward compatibility is not guaranteed.
- The Helm upgrade command does not work.
- Or if we change the values without implement backward compatibility. Because, let say you have a release, and you perform an upgrade, if the version you are upgrading to has changed some parameters and you are not setting the new ones, your new release could end with fewer things than the ones you were expecting.

Also, this PR is introducing a big amount of changes to the Chart. Checking the values.yml it seems that the parameters are all the same, but there are changes in their default values.  For example, psp.create is being changed from `false` to `true` by default. If we guaranteed that the release will be the same as before an upgrade, we can bump just the minor version. But, in any case, IMHO I think a patch is not enough for these changes.

Reply:
@miguelaeh ok then I will bump the major version ^^

Reply:
Bumped it now to version `2.0.0`